target/linux/ath79/dts/qca9531_comfast_cf-e355ac-v2.dts

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "comfast,cf-e355ac-v2", "qca,qca9531";
+	model = "COMFAST CF-E355AC V2";
+
+	aliases {
+		serial0 = &uart;
+		label-mac-device = &eth1;
```

Comment:
&eth1 is 0x0 +1, so probably eth0 on the running device. Am I correct and this is wrong?